engine/src/main/java/org/operaton/bpm/engine/impl/pvm/process/TransitionImpl.java

```diff
@@ -57,21 +57,30 @@ public void setDestination(ActivityImpl destination) {
     destination.getIncomingTransitions().add(this);
   }
 
-  @Deprecated
+  /**
+   * @deprecated Use {@link #addListener(ExecutionListener.EVENTNAME_TAKE, executionListener)} instead
+   */
+  @Deprecated(forRemoval = true)
   public void addExecutionListener(ExecutionListener executionListener) {
     super.addListener(ExecutionListener.EVENTNAME_TAKE, executionListener);
   }
 
+  /**
+   * @deprecated Use {@link #getListeners(ExecutionListener.EVENTNAME_TAKE)} instead
+   */
   @SuppressWarnings({ "rawtypes", "unchecked" })
   @Deprecated
   public List<ExecutionListener> getExecutionListeners() {
```

Comment:
What is the reason to not mark this method `forRemoval=true`?

Reply:
I missed that, good catch

Reply:
done